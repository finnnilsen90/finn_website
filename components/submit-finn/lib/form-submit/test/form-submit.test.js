const React = require('react');
const ReactTestUtils = require('react-dom/test-utils');
const renderer = require('react-test-renderer')
let { shallow, mount, render } = require('enzyme');

const Form = require('../form-submit.js');

let newForm = new Form();

describe('component connection', () => {

    it('should be defined', () => {
        expect(Form).toBeDefined();
    })

})  
    
describe('<Form />', () => {

    const wrapper = render(<Form />);

    it('renders components', () => {
        expect(wrapper).toMatchSnapshot();
    });


})