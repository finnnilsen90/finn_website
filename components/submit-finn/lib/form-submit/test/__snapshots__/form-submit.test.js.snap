// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Form /> renders components 1`] = `
<div
  class="form-submit_Container"
/>
`;
